ckan/adx_config.ini

```diff
@@ -121,15 +121,15 @@ ckan.redis.url = redis://redis:6379/1
 #		Add ``resource_proxy`` to enable resorce proxying and get around the
 #		same origin policy
 # Note: Plugins to the left take precendence over plugins to the right!!! Opposite to what you might expect.
-ckan.plugins = external_storage unaids  emailasusername restricted authz_service versions stats text_view image_view
+ckan.plugins = scheming_datasets external_storage unaids emailasusername restricted authz_service versions stats text_view image_view
```

Comment:
@tomeksabala This is important to double check won't cause any unexpected issues